SECP-service/src/main/resources/assets/app/js/app.js

```diff
@@ -21,6 +36,7 @@ angular.module('SECP', ['ngResource', 'ngRoute', 'ui.bootstrap', 'ui.date', 'rou
         templateUrl: 'views/chat/chats.html',
         controller: 'ChatController',
         css: 'css/chat.css',
+        requiresLogin: true
```

Comment:
This is neat. Is this attribute built into angular-routes?

Reply:
No this attribute is built into angular jwt. :)